<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPanel</class>
 <widget class="QWidget" name="ModifyPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="fieldsTableWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>19</number>
     </attribute>
     <column>
      <property name="text">
       <string>Поле</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Значение</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item row="5" column="2">
    <widget class="QWidget" name="buttonsPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="repeatButton">
        <property name="toolTip">
         <string>F4</string>
        </property>
        <property name="text">
         <string>Повтор</string>
        </property>
        <property name="icon">
         <iconset resource="ui/tracks.qrc">
          <normaloff>:/images/refresh-24.png</normaloff>:/images/refresh-24.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveChangesButton">
        <property name="toolTip">
         <string>Ctrl+Return</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="ui/tracks.qrc">
          <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Сброс</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionApplyChanges">
   <property name="icon">
    <iconset resource="ui/tracks.qrc">
     <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Применить все изменения к выделенным запросам</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionRevertChanges">
   <property name="icon">
    <iconset resource="ui/tracks.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Отменить все изменения</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionRepeatChanges">
   <property name="icon">
    <iconset resource="ui/tracks.qrc">
     <normaloff>:/images/refresh-24.png</normaloff>:/images/refresh-24.png</iconset>
   </property>
   <property name="text">
    <string>Повторить предыдущие изменения</string>
   </property>
   <property name="toolTip">
    <string>Повторить предыдущие изменения</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionRevertField">
   <property name="text">
    <string>Отменить изменение в поле</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionClearField">
   <property name="text">
    <string>Очистить поле</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ui/tracks.qrc"/>
 </resources>
 <connections/>
</ui>
